CMakeLists.txt

```diff
@@ -29,8 +29,8 @@ list(APPEND CMAKE_MODULE_PATH ${BCOS_FRAMEWORK_CMAKE_DIR})
 # init hunter
 include(HunterGate)
 HunterGate(
-    URL "https://github.com/FISCO-BCOS/hunter/archive/da73ad5a35aaee05b9feb77671c744487979c611.tar.gz"
-    SHA1 "9bfa9577907f31a2e064a986c423651dd29e2a45"
+    URL "https://github.com/cyjseagull/hunter/archive/f8f1332f6e156831f97ff99df0dde1825b5fc4cb.tar.gz"
```

Comment:
please use official hunter as possible you can, we should all depend on the official  hunter